### PR TITLE
Fix: add default uniq cursor fields for transction api

### PR DESCRIPTION
**Pull request by Naupio**
problem:
when use calling the transcation api with sorter `[:block_number]` the paginator may return unexpect results with duplicate value of previous page.

this pr solve:
add default implicit unique sorter feilds for paginator to comfirm the returning cursor‘s uniqueness

example:
if call transaction api with sorter [:block_number],we will change it to [:block_number, :hash] to comfirm the uniqueness

### CODE REVIEW

File: lib/godwoken_explorer/graphql/resolovers/transaction.ex [hunk 1]
```diff
@@ -233,6 +233,11 @@ defmodule GodwokenExplorer.Graphql.Resolvers.Transaction do
 
     if sorter do
       order_params = cursor_order_sorter(sorter, :order, @sorter_fields)
```
Comment (zmcNotafraid) [inline lib/godwoken_explorer/graphql/resolovers/transaction.ex:235]: 这两个方法看起来一摸一样，可以抽个方法出来

Comment (Naupio) [reply lib/godwoken_explorer/graphql/resolovers/transaction.ex]: updated in  https://github.com/Magickbase/godwoken_explorer/pull/1012/commits/cf8c409288235eb6ba438a334e38837954fb5445

**Review by Keith-CY — APPROVED** [review]

**Comment by Keith-CY** [comment]
Test case should be added

**Review by zmcNotafraid — APPROVED** [review]

**Comment by Naupio** [comment]
testing case was adding

> Test case should be added

**Pull request merged**